persistence/sql/persister_nid_test.go

```diff
@@ -1777,46 +1778,15 @@ func (s *PersisterTestSuite) TestRevokeRefreshToken() {
 			request.Session = &oauth2.Session{DefaultSession: &openid.DefaultSession{Subject: "sub"}}
 
 			signature := uuid.Must(uuid.NewV4()).String()
-			require.NoError(t, r.Persister().CreateRefreshTokenSession(s.t1, signature, request))
+			require.NoError(t, r.Persister().CreateRefreshTokenSession(s.t1, signature, "", request))
 
 			actual := persistencesql.OAuth2RequestSQL{Table: "refresh"}
 
 			require.NoError(t, r.Persister().RevokeRefreshToken(s.t2, request.ID))
 			require.NoError(t, r.Persister().Connection(context.Background()).Find(&actual, signature))
 			require.Equal(t, true, actual.Active)
 			require.NoError(t, r.Persister().RevokeRefreshToken(s.t1, request.ID))
-			require.NoError(t, r.Persister().Connection(context.Background()).Find(&actual, signature))
-			require.Equal(t, false, actual.Active)
-		})
-	}
-}
-
-func (s *PersisterTestSuite) TestRevokeRefreshTokenMaybeGracePeriod() {
-	t := s.T()
-	for k, r := range s.registries {
-		t.Run(k, func(t *testing.T) {
-			client := &client.Client{ID: "client-id"}
-			require.NoError(t, r.Persister().CreateClient(s.t1, client))
-
-			request := fosite.NewRequest()
-			request.Client = &fosite.DefaultClient{ID: "client-id"}
-			request.Session = &oauth2.Session{DefaultSession: &openid.DefaultSession{Subject: "sub"}}
-
-			signature := uuid.Must(uuid.NewV4()).String()
-			require.NoError(t, r.Persister().CreateRefreshTokenSession(s.t1, signature, request))
-
-			actual := persistencesql.OAuth2RequestSQL{Table: "refresh"}
-
-			store, ok := r.Persister().(*persistencesql.Persister)
-			if !ok {
-				t.Fatal("type assertion failed")
-			}
-
-			require.NoError(t, store.RevokeRefreshTokenMaybeGracePeriod(s.t2, request.ID, signature))
-			require.NoError(t, r.Persister().Connection(context.Background()).Find(&actual, signature))
-			require.Equal(t, true, actual.Active)
-			require.NoError(t, store.RevokeRefreshTokenMaybeGracePeriod(s.t1, request.ID, signature))
-			require.NoError(t, r.Persister().Connection(context.Background()).Find(&actual, signature))
+			require.ErrorIs(t, r.Persister().Connection(context.Background()).Find(&actual, signature), sql.ErrNoRows)
```

Comment:
Token revokation now removes the full token chain from the DB so we just get err no rows here.